docs/tutorial.rst

```diff
@@ -385,12 +363,10 @@ proxy like NGINX or using CDN services.
 
 But for development handling static files by aiohttp server is very convenient.
 
-Fortunately it can be done easy by single call::
-
-    app.router.add_static('/static/',
-                          path=str(project_root / 'static'),
-                          name='static')
+Fortunately it can be done easy by single call:
 
+.. literalinclude:: ../demos/polls/aiohttpdemo_polls/routes.py
+  :pyobject: setup_static_routes
```

Comment:
I like how you split the code, but JFYI there's option to cherry-pick certain lines from the linked file.

Let's check whether @asvetlov is okay with this.

Reply:
I was trying to avoid possible issues with function reordering or new imports. Pointing to the function makes it more robust.

Reply:
I'm ok.